cmake/modules/ddr/GenerateStub.cmake

```diff
@@ -23,18 +23,22 @@ if(NOT input_file)
 	message(FATAL_ERROR "No input file")
 endif()
 
-execute_process(COMMAND grep -E -q "@ddr_(namespace|options):" ${input_file} TIMEOUT 2 RESULT_VARIABLE rc)
+if(NOT EXISTS "${input_file}")
+	message(FATAL_ERROR "Input file '${input_file}' does not exist")
+endif()
+
+execute_process(COMMAND grep -E -q "@ddr_(namespace|options):" ${input_file} RESULT_VARIABLE rc)
 
-if(rc)
+if(rc NOT EQUAL 0)
```

Comment:
It would be better for all concerned if you actually tested things before sharing.
This isn't a legal expression; it must be written `if(NOT rc EQUAL 0)`.